src/DI/RedisExtension.php

```diff
@@ -150,9 +152,20 @@ public function afterCompile(ClassType $class): void
 	{
 		$config = $this->config;
 
+		// Add panel to tracy bar only if debug is allowed
 		if ($config->debug && $config->connection !== []) {
-			$initialize = $class->getMethod('initialize');
-			$initialize->addBody('$this->getService(?)->addPanel($this->getService(?));', ['tracy.bar', $this->prefix('panel')]);
+			$generator = new PhpGenerator($this->getContainerBuilder());
+			$this->initialization->addBody($generator->formatPhp('?;', [
```

Comment:
Why is this needed?

Reply:
This is needed in order to allow debug be set via ENV variable too... The given ENV node (Nette\DI\Statement) needs to be evaluated on the runtime properly (just passing it as another placeholder into the previous statement did not work as it did not unfolded from the Nette\DI\Statement.

Reply:
I see. Thanks, I must take a look at it closer.